### DISCUSSION TITLE
Issues with Accessing GitHub Teacher Toolbox Swag Offer

### DISCUSSION THREAD

**Discussion by hasidudissanayake** [community/community#47852] [GitHub Education] [answered]
Dear Community Members,

I am writing to seek your advice and assistance regarding my issue with accessing the GitHub Teacher Toolbox Swag Offer form. As a mentor at Esoft Metro Campus, Sri Lanka, I am unable to answer the question "How many students do you have?" on the form, and I cannot see the text box to enter the information. (Meanwhile, I have already created a support ticket on this issue: Unable to Answer GitHub Teacher Toolbox Swag Offer #2020387)

I am wondering if any other teachers have faced similar issues when requesting GitHub-branded items and learning materials through the Teacher Toolbox. If so, how were you able to resolve the problem?

I would be grateful if you could share your insights and suggestions on this matter, as our students are eagerly waiting for these resources to enhance their learning experiences.

Thank you for your time and assistance. We appreciate any help we can get to resolve this issue and make the most of the GitHub Teacher Toolbox Swag Offer.

![Screenshot (160)](https://user-images.githubusercontent.com/121795747/219830374-13ec2e73-aede-4ae0-a41a-af020af8a53f.png)

Best regards,
D. M. H. M. Dissanayake (E133351)
Esoft Metro Campus - Sri Lanka

**Comment by samus-aran** [comment, marked as answer]
Hi @hasidudissanayake   👋 

The GitHub Education team apologise for the delay on processing the current Teacher's swag due to a third party bug. We are working through the requests so we appreciate your patience and understanding. The update can be [found here](https://github.com/orgs/community/discussions/46694#discussioncomment-4999443).

**Reply by hasidudissanayake** [reply]
Dear @samus-aran,

Thank you for taking the time to respond to my question about the GitHub Teacher's swag offer. I appreciate the update and understanding that the delay is due to a third-party bug. I will keep an eye on the update and patiently wait for the issue to be resolved.

Thanks again for your response and for your efforts in making the GitHub Education program a great resource for educators.

Best regards,
D. M. H. M. Dissanayake (E133351)
Esoft Metro Campus - Sri Lanka

**Reply by mzunnurainhussain** [reply]
I am unable to locate the link for github teacher toolbox swag offer

**Comment by vanshdhawaneclipe** [comment]
still not fixed
